v5/core/unmarshal_v2_primitives_test.go

```diff
@@ -50,13 +50,15 @@ func TestUnmarshalPrimitiveString(t *testing.T) {
 		"bad_type":  true,
```

Comment:
Should there be a variation in this test for `[][]string`?  That is supposed to work, right?